docs/dev/stream/queryable_state.md

```diff
@@ -208,10 +208,16 @@ Once used in a job, you can retrieve the job ID and then query any key's current
 
 {% highlight java %}
 final Configuration config = new Configuration();
-config.setString(ConfigConstants.JOB_MANAGER_IPC_ADDRESS_KEY, queryAddress);
-config.setInteger(ConfigConstants.JOB_MANAGER_IPC_PORT_KEY, queryPort);
+config.setString(JobManagerOptions.ADDRESS, queryAddress);
+config.setInteger(JobManagerOptions.PORT, queryPort);
 
-QueryableStateClient client = new QueryableStateClient(config);
+final HighAvailabilityServices highAvailabilityServices =
+      HighAvailabilityServicesUtils.createHighAvailabilityServices(
+           config,
+           TestingUtils.defaultExecutor(),
+           HighAvailabilityServicesUtils.AddressResolution.NO_ADDRESS_RESOLUTION);
```

Comment:
Better to do `TRY_ADDRESS_RESOLUTION` here so that we fail fast in case the JM is not reachable from the node we are currently running on.